src/Phing/DropCacheTablesTask.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace DrupalProject\Phing;
+
+require_once 'phing/tasks/ext/pdo/PDOTask.php';
+
+/**
+ * Phing task to drop Drupal's cache tables from the database.
+ */
+class DropCacheTablesTask extends \PDOTask {
```

Comment:
Drop OR Truncate. I see that it's truncate. Let's use everywhere the correct term: truncate.